tests/test_KD_Lib.py

```diff
@@ -150,4 +151,37 @@ def test_attention():
 
     att.train_teacher(epochs=0,plot_losses=False,save_model=False)
     att.train_student(epochs=0,plot_losses=False,save_model=False)
-    att.evaluate(teacher=False)
\ No newline at end of file
+    att.evaluate(teacher=False)
+
+
+def test_NoisyTeacher():
+    teacher_params = [4, 4, 8, 4, 4]
+    student_params = [4, 4, 4, 4, 4]
+    teacher_model = ResNet50(teacher_params, 1, 10)
+    student_model = ResNet18(student_params, 1, 10)
+
+    train_loader = torch.utils.data.DataLoader(
+            datasets.MNIST('mnist_data', train=True, download=True,
+                           transform=transforms.Compose([
+                               transforms.ToTensor(),
+                               transforms.Normalize((0.1307,), (0.3081,))
+                           ])), batch_size=32, shuffle=True)
+
+    test_loader = torch.utils.data.DataLoader(
+            datasets.MNIST('mnist_data', train=False,
+                           transform=transforms.Compose([
+                                   transforms.ToTensor(),
+                                   transforms.Normalize((0.1307,), (0.3081,))
+                               ])),
+            batch_size=32, shuffle=True)
+
+    t_optimizer = optim.SGD(teacher_model.parameters(), 0.01)
+    s_optimizer = optim.SGD(student_model.parameters(), 0.01)
+
+    experiment = NoisyTeacher(teacher_model, student_model, train_loader,
+                              test_loader, t_optimizer, s_optimizer,
+                              alpha=0.4, noise_variance=0.2, device='cpu')
+
+    experiment.train_teacher(epochs=0)
+    experiment.train_student(epochs=0)
```

Comment:
add `save_model=False` to both `train_teacher()` and `train_student()` travis doesn't create directories as such.